src/broadphase/broadphase_interval_tree.cpp

```diff
@@ -178,9 +179,9 @@ void IntervalTreeCollisionManager::setup()
 {
   if(!setup_)
   {
-    std::sort(endpoints[0].begin(), endpoints[0].end(), boost::bind(&EndPoint::value, _1) < boost::bind(&EndPoint::value, _2));
-    std::sort(endpoints[1].begin(), endpoints[1].end(), boost::bind(&EndPoint::value, _1) < boost::bind(&EndPoint::value, _2));
-    std::sort(endpoints[2].begin(), endpoints[2].end(), boost::bind(&EndPoint::value, _1) < boost::bind(&EndPoint::value, _2));
+    std::sort(endpoints[0].begin(), endpoints[0].end());
+    std::sort(endpoints[1].begin(), endpoints[1].end());
+    std::sort(endpoints[2].begin(), endpoints[2].end());
```

Comment:
This is actually not a C++11-related change. (Well, sort of, s/boost/std/g didn't work.) The simplification is because of the operator< declaration in the Endpoint class.